packages/data/src/plugins/action-types.ts

```diff
@@ -0,0 +1,12 @@
+enum ACTION_TYPES {
+	UPDATE_ACTIVE_PLUGINS = 'UPDATE_ACTIVE_PLUGINS',
+	UPDATE_INSTALLED_PLUGINS = 'UPDATE_INSTALLED_PLUGINS',
+	SET_IS_REQUESTING = 'SET_IS_REQUESTING',
+	SET_ERROR = 'SET_ERROR',
+	UPDATE_JETPACK_CONNECTION = 'UPDATE_JETPACK_CONNECTION',
+	UPDATE_JETPACK_CONNECT_URL = 'UPDATE_JETPACK_CONNECT_URL',
+	SET_PAYPAL_ONBOARDING_STATUS = 'SET_PAYPAL_ONBOARDING_STATUS',
+	SET_RECOMMENDED_PLUGINS = 'SET_RECOMMENDED_PLUGINS',
+}
+
+export default ACTION_TYPES;
```

Comment:
Can we avoid default exports? naming them avoids confusion and has better IDE support.